media-libs/embree/files/embree-3.13.5-fix-arm64.patch

```diff
@@ -0,0 +1,48 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
```

Comment:
Link to the commit source + any relevant gentoo bugs too, pls (at the top of the patch)